lib/fast_jsonapi/serialization_core.rb

```diff
@@ -157,12 +156,13 @@ def get_included_records(record, includes_list, known_included_objects, params =
             included_objects = [included_objects] unless relationship_type == :has_many
 
             included_objects.each do |inc_obj|
+              serializer = (inc_obj.class.to_s + "Serializer").constantize if @relationships_to_serialize[item][:polymorphic].is_a?(Hash)
```

Comment:
this would fail when there are name spaces. Using this approach might take care of this https://github.com/Netflix/fast_jsonapi/blob/f4ad6bc2ac2bfde87f6b989e50647f8b335db518/lib/fast_jsonapi/object_serializer.rb#L218

Reply:
@krzkrzkrz  these comments need to be addressed to start with.

Reply:
@shishirmk In the interim. Is there an alternative approach we can use? Polymorphic includes is actually whats hindering us right now from using this library. Related to what I posted here https://github.com/Netflix/fast_jsonapi/issues/266